cse/indexing/FileIndexer.py

```diff
@@ -178,7 +179,10 @@ def __processComment(self, index, cid, comment):
     )
 
     start = time.process_time()
-    FileIndexer("data", "comments.data", "articleMapping.data", "authorMapping.data", prep).index()
+    FileIndexer(".", "comments.csv", "articles.csv", "authors.csv", prep).indexPostingList()
```

Comment:
may you also change the `indexPostingList()` method to build the new document map?

Reply:
done